src/actions/actions.js

```diff
@@ -11,9 +11,10 @@ export const getTeams = () => async dispatch => {
 };
 
 export const getSelectedTeam = team => async dispatch => {
+
     dispatch({ type: 'GET_SELECTED_TEAM', payload: team });
     dispatch({ type: 'GET_TEAM_COLOR', payload: team.primaryColor });
-    dispatch({ type: 'PRELOAD_PLAYER', payload: null });
+    dispatch({ type: 'PRELOAD_PLAYER_DETAILS', payload: null });
```

Comment:
You don't need to dispatch this here. We'll dispatch it in the getSelectedPlayer action.